zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -138,11 +146,131 @@ public static Either<ProblemDetail, ProcessInstanceSort> toSearchQuerySort(
     return Either.right(null);
   }
 
+  public static Either<ProblemDetail, UserTaskSort> toUserTaskSearchQuerySort(
+      final List<SearchQuerySortRequest> sorting) {
+    if (sorting != null && !sorting.isEmpty()) {
+      final var builder = SortOptionBuilders.userTask();
+
+      for (final SearchQuerySortRequest sort : sorting) {
+        final var field = sort.getField();
+        final var order = sort.getOrder();
+
+        if ("creationTime".equals(field)) {
+          builder.creationDate();
+        } else if ("completionTime".equals(field)) {
+          builder.completionDate();
+        } else {
+          throw new RuntimeException("unkown sortBy " + field);
```

Comment:
🔧 You could use this case as an input validation error and return an `Either.Left` with a `ProblemDetail`. This can then be handled by the caller, e.g. the controller. That's how we do input validation in the other user task mapping methods here, for example.